api/module.mk

```diff
@@ -54,6 +57,13 @@ $(INTEGRATION): $(TEST_SCM_SRC) $(TEST_SCM_RESOURCES) $(CLOJURE_PROJECT)
 	$(TEE) $(DERIVED_MODULE_DIR)/integration.log
 	@$(TOUCH) $@
 
+.PHONY: SYSTEM_TESTS
+$(SYSTEM): $(SYSTEM_TESTS)
+	@$(TOUCH) $@
+$(SYSTEM_TESTS): $(TEST_SCM_SRC) $(TEST_SCM_RESOURCES) $(CLOJURE_PROJECT)
+	$(EXPORT) CPCACHE=$(CPCACHE_DIR);            \
+    $(CLOJURE) $(CLJFLAGS) -A:test system --focus wfl.system.v1-endpoint-test/$@
+
```

Comment:
This works too, this just sorta forces us to keep a list of exact system test names outside of our code. Also it feels funky to me to have to separately stand up a server and liquibase and stuff to run this. I bet this also breaks the `all` thing, because it can't run straight through now. I think I could start a server from make but i dunno how I would stop it